src/main/java/nextstep/subway/domain/line/LineController.java

```diff
@@ -0,0 +1,63 @@
+package nextstep.subway.domain.line;
+
+import nextstep.subway.domain.section.dto.SectionCreateRequest;
+import nextstep.subway.domain.section.dto.SectionResponse;
+import nextstep.subway.domain.line.dto.LineCreateRequest;
+import nextstep.subway.domain.line.dto.LineResponse;
+import nextstep.subway.domain.line.dto.LineUpdateRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+    private LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineCreateRequest request) {
+        LineResponse line = lineService.saveLine(request);
+        return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
+    }
+
+    @GetMapping("/lines")
+    public ResponseEntity<List<LineResponse>> findAllLines() {
+        List<LineResponse> lines = lineService.findLines();
+        return ResponseEntity.ok(lines);
+    }
+
+    @GetMapping("/lines/{id}")
+    public ResponseEntity<LineResponse> findLineById(@PathVariable Long id) {
+        LineResponse line = lineService.findLineById(id);
+        return ResponseEntity.ok(line);
+    }
+
+    @PutMapping("/lines/{id}")
+    public ResponseEntity<LineResponse> updateLine(@PathVariable Long id, @RequestBody LineUpdateRequest lineRequest) {
+        lineService.updateLine(id, lineRequest);
+        return ResponseEntity.ok().build();
+    }
+
+    @DeleteMapping("/lines/{id}")
+    public ResponseEntity<?> deleteLine(@PathVariable Long id) {
```

Comment:
`ResponseEntity<Void>`를 반환함으로써, HTTP 응답 본문에 데이터가 없음을 명시적으로 표현해보는 건 어떨까요?
```suggestion
    public ResponseEntity<Void> deleteLine(@PathVariable Long id) {
```

Reply:
항상 기계적으로 <?>를 붙였었는데, 이번 기회에 Void를 적극적으로 사용해야겠네요!